src/leafdoc.js

```diff
@@ -471,7 +471,7 @@ Leafdoc.prototype.addStr = function(str, isSource) {
 				} else if (directive === 'param') {
 					// Params are param name, type.
 					/// TODO: Think about default values, or param explanation.
-					var split = content.split(',');
+					var split = content.split(':');
```

Comment:
This code works only for `@param foo: Type` constrcuctions, so it should not affect parsing of functions